lib/pipeline.ml

```diff
@@ -89,22 +110,8 @@ let docker_specs ~analysis =
             ~analysis (`Lint `Fmt)
           :: Spec.opam_monorepo builds
       | `Opam_build selections ->
-          let selections =
-            (* TODO: This filter will be removed when a new version of debian in which capnproto has this fix "https://github.com/capnproto/capnproto/pull/1830".
```

Comment:
The referenced issue was resolved last year: https://github.com/capnproto/capnproto/pull/1830